test/plugins/features/movie/controller.test.js

```diff
@@ -13,12 +13,49 @@ describe('movie controller', () => {
       return Controller.create(payload)
       .then((movie) => {
         expect(movie.get('title')).to.eql(payload.title);
+      });
+    });
 
-        return new Movie({ id: movie.id }).fetch();
-      })
-      .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
-        expect(movie.get('release_year')).to.eql(payload.release_year);
+  });
+
+  describe('get', () => {
```

Comment:
Since `get` could refer to either the list or retrieve, it's usually better to name these `describe`s by the function name that is being tested. So in this case, it should be `describe('list', () => {`.